aim/sdk/adapters/acme.py

```diff
@@ -0,0 +1,73 @@
+from typing import Optional, Dict
+from aim.sdk.run import Run
+from aim.ext.resource.configs import DEFAULT_SYSTEM_TRACKING_INT
+from acme.utils.loggers.base import Logger, LoggingData
+
+
+class AimCallback:
+    def __init__(
+        self,
+        repo: Optional[str] = None,
+        experiment_name: Optional[str] = None,
+        system_tracking_interval: Optional[int] = DEFAULT_SYSTEM_TRACKING_INT,
+        log_system_params: bool = True,
+        args: Optional[Dict] = None,
+    ):
+        self.repo = repo
+        self.experiment_name = experiment_name
+        self.system_tracking_interval = system_tracking_interval
+        self.log_system_params = log_system_params
+        self._run = None
+        self._run_hash = None
+
+        self.setup(args)
+
+    @property
+    def experiment(self):
+        if not self._run:
+            self.setup()
+        return self._run
+
+    def setup(self, args=None):
+        if not self._run:
+            if self._run_hash:
+                self._run = Run(
+                    self._run_hash,
+                    repo=self.repo,
+                    system_tracking_interval=self.system_tracking_interval,
+                    log_system_params=self.log_system_params,
+                )
+            else:
+                self._run = Run(
+                    repo=self.repo,
+                    experiment=self.experiment_name,
+                    system_tracking_interval=self.system_tracking_interval,
+                    log_system_params=self.log_system_params,
+                )
+                self._run_hash = self._run.hash
+
+        if args:
+            for key, value in args.items():
+                self._run.set(key, value, strict=False)
+
+    def track(self, logs, context, step=None):
+        self._run.track(logs, step=step, context=context)
+
+    def close(self):
+        if self._run and self._run.active:
+            self._run.close()
+
+
+class AimWritter(Logger):
```

Comment:
I guess this should have been an `AimWriter`, right? 
Please also make sure to fix the mentions in the docs and examples as well.